src/buckets/api/index.ts

```diff
@@ -0,0 +1,31 @@
+import {normalize} from 'normalizr'
+
+import {getBuckets} from 'src/client'
+import {fetchDemoDataBuckets} from 'src/cloud/apis/demodata'
+
+import {arrayOfBuckets} from 'src/schemas'
+import {Bucket, BucketEntities} from 'src/types'
+
+import {CLOUD} from 'src/shared/constants'
+import {BUCKET_LIMIT} from 'src/resources/constants'
+
+export const fetchAllBuckets = async (orgID: string, limit = BUCKET_LIMIT) => {
```

Comment:
this method isn't a thunk, like the comment says, it doesn't belong in the thunks file